docs/conveyor/user-guide.md

```diff
@@ -0,0 +1,49 @@
+# Getting started
+## Preparation
+At first, you need to get some DAI and USDC tokens, the following steps show how to get these tokens on the Ethereum Ropsten testnet
```

Comment:
You need to make sure that your wallet is funded with some DAI and USDC tokens. If you don't have the necessary tokens yet, follow the steps below to get funded.